client/components/happychat/connection-connected.jsx

```diff
@@ -11,8 +9,7 @@ export default connect(
 	( state, ownProps ) => ( {
 		getAuth: ownProps.getAuth || getHappychatAuth( state ),
 		isConnectionUninitialized: isHappychatConnectionUninitialized( state ),
-		isHappychatEnabled:
-			config.isEnabled( 'happychat' ) && ! shouldShowHelpCenterToUser( getCurrentUserId( state ) ),
+		isHappychatEnabled: config.isEnabled( 'happychat' ),
```

Comment:
This will bring back the bug explained [here](https://github.com/Automattic/wp-calypso/pull/66407#:~:text=again.%20Interestingly%2C%20it%20seems%20the-,remnants,-of%20FAB%20are%20active%20and). 

Let's copy the solution from [here](https://github.com/Automattic/wp-calypso/pull/68046/files#diff-52162a5dc985b44902901411edeb96565c4c4be47adc7e8f32ffaff072d4b3a5R15-R17)

Then pass this [this](https://github.com/Automattic/wp-calypso/pull/68046/files#diff-bd7ff7b9859f24ed9347c9c12b799ddb65f3a77c0ca58f8bd7bfeca923b17b32R163).

Reply:
👍